secscanner2junit/container_scanning.py

```diff
@@ -0,0 +1,40 @@
+from junit_xml import TestSuite, TestCase
+
+from secscanner2junit.config import Config
+from secscanner2junit.parser import Parser
+from secscanner2junit.vulnerability import ContainerScanningVulnerability
+
+
+# See following links to learn more about containers scanners and theirs output
+# Example https://gitlab.com/gitlab-examples/security/security-reports/-/blob/master/samples/container-scanning.json
+# https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/blob/master/dist/container-scanning-report-format.json
+class ContainerScanningParser(Parser):
```

Comment:
Once this PR is merged, I'll update the SAST and Secrets parsers to also employ your new structure (if thats fine for you?)

I guess after that we can refactor out the `parse` method into the `Parser`. Not sure about the `parse_vulnerability` method atm.

Reply:
that was also my idea to do it this way (lets give some time f.e a week to use this container-scanning to see if there will be some problems or not)